cni-plugin/deployment/scripts/install-cni.sh

```diff
@@ -298,6 +304,11 @@ monitor() {
           cni_conf_sha="$(sha256sum "$directory/$filename" | while read -r s _; do echo "$s"; done)"
         fi
       fi
+    done &
+  inotifywait -m "/var/run/secrets/kubernetes.io/serviceaccount/token" -e create,delete,moved_to,modify |
```

Comment:
In `create_cni_conf()` we have `SERVICE_ACCOUNT_PATH=/var/run/secrets/kubernetes.io/serviceaccount`. Can you make that a global variable, so we can reuse it here?